compiler/rustc_metadata/src/rmeta/encoder.rs

```diff
@@ -873,14 +873,14 @@ fn should_encode_attrs(def_kind: DefKind) -> bool {
         | DefKind::AssocConst
         | DefKind::Macro(_)
         | DefKind::Field
-        | DefKind::Impl { .. } => true,
+        | DefKind::Impl { .. }
+        | DefKind::ConstParam => true,
```

Comment:
do we need to encode const param attrs in metadata? I'd assume we'd get all the information from `ty::Generics` if it's from an upstream crate